client/profile-wizard/steps/store-details.js

```diff
@@ -167,67 +177,110 @@ class StoreDetails extends Component {
 		return (
 			<Fragment>
 				<H className="woocommerce-profile-wizard__header-title">
-					{ __( 'Where is your store based?', 'woocommerce-admin' ) }
+					{ __( 'Welcome to WooCommerce', 'woocommerce-admin' ) }
 				</H>
 				<H className="woocommerce-profile-wizard__header-subtitle">
 					{ __(
-						'This will help us configure your store and get you started quickly',
+						"Tell us about your store and we'll get you set up in no time",
 						'woocommerce-admin'
 					) }
-				</H>
 
-				<Card>
-					<Form
-						initialValues={ this.initialValues }
-						onSubmitCallback={ this.onSubmit }
-						validate={ validateStoreAddress }
+					<Tooltip
+						text={ __(
+							'Your store address will help us configure currency\n options and shipping rules automatically.\n This information will not be publicly visible and can\n easily be changed later.',
```

Comment:
I used newlines and whitespace: pre to get this displaying as close to the designs as possible. This seems problematic for translations?

I tried modifying elements on popover more with styles, but it was quite constrained, I could not retain the exact width of the design.